hackIDE/static/hackIDE/js/custom.js

```diff
@@ -411,6 +411,17 @@ $(document).ready(function(){
 							$(".error-message").html(response.run_status.stderr);
 						}
 					}
+					// To handle case when hackerearth api doesn't return a valid response.
```

Comment:
@adi8 yes, I was referring to the case when the server crashes and doesn't return any response within the timeout limit.